addons/kubearmor/disable

```diff
@@ -1,8 +1,9 @@
 #!/usr/bin/env bash
 
+
 echo "Removing kubearmor from k8s cluster"
 
-sudo microk8s karmor uninstall
+microk8s.helm uninstall kubearmor-operator -n kubearmor
```

Comment:
Please call the helm command as shown in https://github.com/canonical/microk8s-community-addons/blob/main/addons/cilium/enable

Reply:
@ktsakalozos Could you please review the change now?